week-2-practise/level-3/6-map.js

```diff
@@ -12,10 +12,14 @@ function tidyUpString(str) {
 
 function captialise(str) {
   // complete this function
+  var myString = str.split("");
+  myString[0] = myString[0].toUpperCase();
+  return myString.join("");
 }
 
 var mentors = ["/Daniel ", "irina ", " Gordon", "ashleigh "];
-var mentorsTidyAndCapitalised;
+var theMentors = mentors.map(tidyUpString);
```

Comment:
This variable holds an array of strings that were tidied up - so what would be a better name for it?